### DISCUSSION TITLE
GL-C-002P not working

### DISCUSSION THREAD

**Discussion by BlueScreenTT** [Koenkk/zigbee2mqtt#17021] [Devices]
Hi

i have some big problems wit a GL-C-002P
it connects then disconnects and then that loop continues.

is the device broken or is there something wrong with Z2M ?

got 30 + other devices on the network.

**Comment by CalvinArcurid** [comment]
Sorry to hear you're having trouble with your GL-C-002P. It's hard to say for sure what the issue might be without more information. Have you tried resetting the device and/or resetting your Zigbee network? That might help resolve the issue.

If that doesn't work, it's possible the device is faulty and might need to be replaced. However, it's also possible that there's an issue with your Zigbee network or your Z2M setup. Have you tried contacting the manufacturer or checking any forums to see if others have had similar issues?

It's great to hear that you have over 30 devices on your network, which means you're doing some cool things with home automation! Let me know if there's anything else I can do to help.

**Comment by BlueScreenTT** [comment]
I have bought a new one so ill try tomorrow again.  It was not
working 1,5 years ago also and I have reinstalled HA and Z2M many times
after that and never gotten it to work.   Ill write here if the new one
works or not

//Thomas

søn. 19. mar. 2023 09.35 skrev Carson Culver ***@***.***>:

**Comment by BlueScreenTT** [comment]
yeah.

i have 6 tasmota, 3 esphome and 38 ZB so there is enough to have fun with :-)

So the 5in1 mini ultra was broken :-(   from factory
the new one is working.

well  it could be that this model with that firmware is not working i dont
know   but the new one is working :-)
old one
Byggdatum för firmware20211223Firmware version10251203
new one Byggdatum för firmware20220915Firmware version10751203


Den søn. 19. mar. 2023 kl. 22.52 skrev Thomas Gjørtz Christensen <
***@***.***>:
